style.css

```diff
@@ -1 +1,56 @@
 /*  Add your styling here */
+body  {
+    display: grid;
+    grid-template-areas: 1fr 2fr 4fr 3r;
+    background-color: #ffe6ff;
+}
+
+.nav-bar > ul {
+   display: inline;
+   padding: 100px 6em 6em 7em;
+   font-size: 1.25em;
+}
+
+.title, .tagline {
+    text-align: center;
+}
+
+.title {
+    font-size: 3em;
+    padding: 1em 1em;
+}
+.tagline {
+    font-size: 2em;
+    padding: .75em;
+}
+
+.order-now {
+    border-radius: 55%;
+    padding-bottom: 2em;
+    font-size: 1em;
+}
+
+.info {
+    text-align: center;
+    background-color: #ff80ff;
+}
+
+.cake-images {
+    display: flex;
+}
+
+.img1 {
+    width: 25%;
```

Comment:
Instead of setting 25% on here (and you've also set it on the HTML), consider instead using flex-basis on the figure elements.

Then size your images to be 100% of whatever that containing figure element is.